README.md

```diff
@@ -25,37 +25,40 @@
 <br />
 
 ## Install Giskard 🐢
-You can install the latest version of Giskard from PyPi using pip :
+Install the latest version of Giskard from PyPi:
```

Comment:
I would say
Install the latest version of Giskard from PyPi using pip:
This clarifies both the source of the Giskard package (PyPI) and the tool being used to perform the installation (pip)